src/api/controllers/routes.controller.js

```diff
@@ -0,0 +1,32 @@
+
+import { createRoute, deleteRoute, getAllRoutes, getRouteById, updateRoute } from "../services/route.service";
+
+export  const getRoutes = (req, res) => {
+    const routes = getAllRoutes;
+    
+    res.send(routes);
+}
+
+export const getRoute = (req, res) => {
+    const route = getRouteById(req.params.id);
+    
+    res.send(route);
+}
+
+export const saveRoute = (req, res) => {
+    const route = createRoute(req.body);
+    
+    res.send(route);
+}
+
+export const routeUpdate = (req, res) => {
+    const route = updateRoute(req.params.id, req.body);
+    
+    res.send(route);
+}
+
+export const routeDelete = (req, res) => {
+    const route = deleteRoute(req.params.id);
+    
+    res.send(route);
+}
```

Comment:
i need you send json files [refer](https://github.com/atlp-rwanda/rca-phantom-team1-bn/pull/10/files#diff-259e2aeb7411d1bdb2045e475b436e668eff745d5fe360df7bd5274d1c927091) to this